packages/ui/src/elements/Upload/index.tsx

```diff
@@ -175,14 +175,19 @@ export const Upload: React.FC<UploadProps> = (props) => {
   const handleUrlSubmit = async () => {
     if (fileUrl) {
       try {
-        const response = await fetch(fileUrl)
-        const data = await response.blob()
+        // Make a request to the Payload 'paste-url' endpoint - /api/paste-url
+        const response = await fetch(`/api/paste-url?src=${encodeURIComponent(fileUrl)}`)
```

Comment:
We need to get the api route from the config, since the user has control of setting a custom api route.

Reply:
Done.